testing/xfpga/test_metadata_c.cpp

```diff
@@ -52,11 +52,9 @@ class metadata_c
     system_->initialize();
     system_->prepare_syfs(platform_);
     ASSERT_EQ(fpgaInitialize(NULL), FPGA_OK);
-    fpga_guid fme_guid;
 
-    ASSERT_EQ(uuid_parse(platform_.devices[0].fme_guid, fme_guid), 0);
     ASSERT_EQ(xfpga_fpgaGetProperties(nullptr, &filter_), FPGA_OK);
-    ASSERT_EQ(fpgaPropertiesSetGUID(filter_, fme_guid), FPGA_OK);
+    ASSERT_EQ(fpgaPropertiesSetDeviceID(filter_, platform_.devices[0].device_id), FPGA_OK);
```

Comment:
Should probably set the VendorID as well with this approach.  VID/DID must be used as a pair.